tests/test_convert_data.py

```diff
@@ -79,8 +79,15 @@ def test_spreadsheet_data_conversion():
 
     result = usecase.convert_spreadsheet("some-id")
     csv = FILE_VALUES[0]
-    expected_row = "The Ukrainian House,Poland,Warsaw,\"ul. Zamenhofa 1, 00-153\",52.24734033,20.9964833," \
-                   "Accommodation,Fundacja “Nasz Wybór”,Crisis support center"
+    expected_row = 'The Ukrainian House,Poland,Warsaw,' \
```

Comment:
Are you using some auto-formatter that converts all quotes to single-quotes?

Reply:
Yeah, one style of quotes does make it a bit easier for me. Do we have a preference what quotes to use?

Reply:
I agree that one style is better, I don't have a preference but I was leaning towards double-quotes because is later we'll enable some auto-formatter like `black` it's going to force double-quotes

Reply:
No problem, I'll go with that from now on then.